### ISSUE TITLE
[Feature Request] A guide for creating custom DeepDream models

### ISSUE THREAD

**Issue by ProGamerGov** [tensorflow/tensorflow#10419]
The current DeepDream guide located [here](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/tutorials/deepdream), uses the inception5h model. From what I can tell, it does not appear very straightforward in terms of how to fine tune the model in order to create different DeepDream hallucinations from a custom data set. It also does not appear to be relatively easy to change the model that the guide uses. 

I think that an additional guide which shows individuals how to fine tune a model for the purposes of DeepDream would be useful for those trying to explore the artistic and visual aspects of TensorFlow models. I haven't been able to find any guide for creating custom DeepDream models in Tensorflow, so I am not sure where to start.

**Comment by jart** [comment]
Thank you for the feedback. Generally speaking, tutorials are meant to cover a breadth of topics. Our goal is to show users what's possible and help them get the ball rolling.

Since the DeepDream notebook is very popular, it might be worth going more in depth. I'm not sure if it's something we have time to work on. But I think it's worth asking @znah what he thinks.

**Comment by ProGamerGov** [comment]
@jart At the moment, I've found it extremely confusing as to how to use other pre-trained models with the tutorial's code. I've tried using the model from: http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz, but I couldn't figure out how to get it working with the code. The stripped Android model (inception5h) seems to be really different from other models, and I haven't yet figured out how to use a different model with the DeepDream tutorial code, let alone a model that I finetuned myself.

**Comment by varun19299** [comment]
Same issue as @ProGamerGov , the input tensors are not recognised for either of inception v3 or any subsequent models (model zoo, tensorflow).

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting TensorFlower: It has been 14 days with no activity and the `awaiting tensorflower` label was assigned. Please update the label and/or status accordingly.

**Comment by jart** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
